### DISCUSSION TITLE
How to obtain the installation path in wxs

### DISCUSSION THREAD

**Discussion by EHW-liao** [tauri-apps/tauri#7088] [Q&A] [answered]
I want to obtain the installation directory selected by the user in the custom wxs file and reference it in the registry write operation when building installation packages using Wix. How can I implement this.

**Comment by FabianLars** [comment, marked as answer]
If you're modifying tauri's `main.wxs` file then the there is a `INSTALLDIR` var. For example here it is used as a registry value: https://github.com/tauri-apps/tauri/blob/dev/tooling/bundler/src/bundle/windows/templates/main.wxs#L112

**Reply by EHW-liao** [reply]
thank you!!!!!!!!!!!!!!!!!
